src/RestaurantButton.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable indent*/
+import React from "react";
+// import Restaurant from "./Restaurant";
+
+function RestaurantButton(props) {
+  return (
+    <button
+      className="add-btn"
+      onClick={props.update}
+      className="btn btn-primary"
```

Comment:
Good to see you passing functions as props, this is a step up from passing simple information, to make this even better try naming your props in a way that you can tell what is does straight away, also try to just add all your classNames in one line to keep you code as readable as possible